README.md

```diff
@@ -18,30 +18,30 @@ This Github template is meant as the starting point for building Botpress Custom
 9. In the modules page, click unpack now next to your module's name. ![](3.png)
 10. Go back up to the list of Stable modules, and activate it by clicking the toggle next to the module's name. ![](4.png)
 11. Start editing a chatbot. notice the flag icon for the custom module.
-12. When you change your code, restart from from step 7.
+12. When you change your code, restart from step 7.
 
 
 ## (Advanced) Where's Hot Reload?
-Botpress doesn't currently support hot reload for custom module development. If you want to make changes and check them often, we recommend you clone the main repo and follow the instructions found [here](https://botpress.com/docs/building-chatbots/developers/custom-modules#local-development-tips). Changes will be reflected upon restarting the botpress server.
+Botpress doesn't currently support hot reload for custom module development. If you want to make changes and check them often, we recommend you clone the main repo and follow the [Local Development Tips (Internet Archive link)](https://web.archive.org/web/20220706031741/https://botpress.com/docs/building-chatbots/developers/custom-modules#local-development-tips). Changes will be reflected upon restarting the Botpress server.
```

Comment:
There were a whole bunch of links in this document to a page that doesn't appear to exist anymore in the docs system (I searched).

I'm not seriously suggesting using Internet Archive Links. This is basically a "fix me" for @adkah